core/lib/qcbor/sub.mk

```diff
@@ -2,9 +2,9 @@ global-incdirs-y += inc
 
 cflags-y += -Wno-declaration-after-statement
 cflags-y += -Wno-redundant-decls
-global-cpp-flags-y += -DQCBOR_DISABLE_FLOAT_HW_USE
-global-cpp-flags-y += -DQCBOR_DISABLE_PREFERRED_FLOAT
-global-cpp-flags-y += -DUSEFULBUF_DISABLE_ALL_FLOAT
+global-cppflags-y += -DQCBOR_DISABLE_FLOAT_HW_USE
+global-cppflags-y += -DQCBOR_DISABLE_PREFERRED_FLOAT
+global-cppflags-y += -DUSEFULBUF_DISABLE_ALL_FLOAT
```

Comment:
These fixes should be squashed in commit "core/lib/qcbor: add build configuration for QCBOR library".

Reply:
DONE